swatch-core/src/main/resources/swatch-core/application.yaml

```diff
@@ -139,6 +139,7 @@ rhsm-subscriptions:
   package_uri_mappings:
     # this mapping required here because it is used by our SecurityConfig, which is shared
     org.candlepin.subscriptions.resteasy: ${PATH_PREFIX}/${APP_NAME}/v1
+    org.candlepin.subscriptions.conduit.resteasy: ${PATH_PREFIX}/${APP_NAME}/v1
```

Comment:
Is it possible to move this to swatch-system-conduit/src/main/resources/application.yaml ? I *think* it should work just as well there, if not fine to keep it here.

Reply:
@kahowell got this update in, could you take a look?Thanks!